versioneer.py

```diff
@@ -606,11 +606,11 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
         if verbose:
             print("keywords are unexpanded, not using")
         raise NotThisMethod("unexpanded keywords, not a git-archive tarball")
-    refs = set(r.strip() for r in refnames.strip("()").split(","))
```

Comment:
I'm not sure it is  good idea to clean up versioneer since this is just a file from another project.  Maybe this should be excluded?

Reply:
it probably doesn't matter much. we already replace *some* formatting code here as well.